Assets/Scripts/CreatureStats.cs

```diff
@@ -224,7 +262,6 @@ public void PerformAttack(CreatureStats target)
         {
```

Comment:
Should the popup on line 258 go after damage is increased on line 259?

Reply:
Yes, that's correct.